docs/onelake/onelake-onecopy-quickstart.md

```diff
@@ -20,6 +20,8 @@ In this guide, you will:
 - Analyze and transform data with Spark using a Fabric notebook.
 
 - Query one copy of data on OneLake with SQL.
+  
+- Please note as for tables inside of Lakehouse: they should be created inside of the Table folder, only, and not using intermediate folders in between.
```

Comment:
This note is out of place here.  I recommend you put it somewhere close to information about creating a table (perhaps step 5 in this doc)?  Is this info already in the data engineering docs? If not, I recommend starting there.  Do you want this note in other tutorials that mention writing to tables too?

Reply:
Hi thanks for this, I am moving the note to the to step 5. It makes sense to alter the databricks/fabric documentation as well. I create a PR for that